lib/pure/collections/sequtils.nim

```diff
@@ -80,29 +80,6 @@ import macros
 when not defined(nimhygiene):
   {.pragma: dirty.}
 
-
```

Comment:
Do not touch this module. The probability that your changes here produce regressions is effectively 100%.

Reply:
This module contains one of the more important changes. It contains the refactoring of ``toSeq`` into a style that I consider sane to use. But I guess before I am allowed to refactor ``toSeq`` I have to prove that there are problems in the current implementation that would be fixed here in this version.

Reply:
> The probability that your changes here produce regressions is effectively 100%

indeed, see regressions I found here: https://github.com/nim-lang/Nim/pull/13687#issuecomment-602307527
even if there were no regressions, there is no reason to bundle these unrelated changes in this PR